modules/programs/process-compose.nix

```diff
@@ -0,0 +1,77 @@
+{ config, lib, pkgs, ... }:
+let
+  inherit (lib) mkEnableOption mkPackageOption mkIf mkOption types;
+
+  cfg = config.programs.process-compose;
+
+  settingsFormat = pkgs.formats.yaml { };
+in {
+  meta.maintainers = [ lib.hm.maintainers.dudeofawesome ];
+
+  options.programs.process-compose = {
+    enable = mkEnableOption "process-compose";
+
+    package = mkPackageOption pkgs "process-compose" { };
+
+    settings = mkOption {
+      type = settingsFormat.type;
+      default = null;
+      description = ''
+        Specify the configuration for process-compose.
+
+        Check [the documentation](https://f1bonacc1.github.io/process-compose/tui/#tui-state-settings) for available options.
+      '';
+    };
+
+    shortcuts = mkOption {
+      type = settingsFormat.type;
+      default = null;
+      description = ''
+        Specify shortcuts for process-compose.
+
+        Check [the documentation](https://f1bonacc1.github.io/process-compose/tui/#shortcuts-configuration) for available options.
+      '';
+    };
+
+    theme = mkOption {
+      type = settingsFormat.type;
+      default = null;
+      description = ''
+        Specify a theme for process-compose.
+
+        Check [the documentation](https://f1bonacc1.github.io/process-compose/tui/#tui-themes) for available options.
+      '';
+    };
+  };
+
+  config = let
+    configDir = if (pkgs.stdenv.targetPlatform.isDarwin) then
+      "Library/Application Support/process-compose/"
+    else
+      "$XDG_CONFIG_HOME/process-compose/";
```

Comment:
It looks like the [linux tests are failing](https://github.com/nix-community/home-manager/actions/runs/13173123866/job/36766898135?pr=5788). I wonder if it could be caused by the `XDG_CONFIG_HOME` var not working here?